.github/workflows/main.yml

```diff
@@ -1,6 +1,7 @@
 name: CI
 
 on:
+  workflow_dispatch:
```

Comment:
Neat! I didn't know about that. Could be handy for all sorts of things, especially with the addition of `inputs:`.